config/locales/it.yml

```diff
@@ -578,8 +579,46 @@ it:
+      licensetext: "<div class=\"box wide\">Tutte le azioni sul sito sono pubblice ad eccezione dei messaggi privati tra utenti. Le azioni di moderazione possono essere consultate sul <a href=\"/moderations\">Log di Moderazione</a>.<br/><a rel=\"license\" href=\"https://creativecommons.org/licenses/by/4.0/deed.it\"><img alt=\"Licenza Creative Commons\" style=\"border-width:0\" src=\"https://i.creativecommons.org/l/by/4.0/88x31.png\" /></a><br /><span xmlns:dct=\"http://purl.org/dc/terms/\" property=\"dct:title\">Gambe.ro</span> è pubblicato secondo i termini della <a rel=\"license\" href=\"https://creativecommons.org/licenses/by/4.0/deed.it\">licenza Creative Commons Attribution 4.0 International</a>.</div>"
+      abouttext: "<p>gambe.ro nasce dalla volontà di un gruppo di programmatori di creare uno spazio di discussione in lingua italiana e di scambio focalizzato intorno ai temi dello sviluppo software. Al contrario di buona parte dei siti, gruppi e chat venuti prima di noi, vogliamo coinvolgere specificatamente e unicamente la comunità di persone che produce software. Crediamo che allargare eccessivamente lo spettro delle tematiche trattate porti alla creazione di una community frammentata in cui convivono persone con background troppo differenti e pochi interessi in comune. Riteniamo che in una community del genere la qualità del dibattito sarebbe inevitabilmente penalizzata
```

Comment:
"focalizzato intorno a" suona male imho

Reply:
Manca il punto in fondo alla frase

Reply:
"focalizzato sullo sviluppo software" potrebbe essere un'alternativa, ma a me "focalizzato intorno a" non dispiace.

Reply:
troppo tempo a parlare in inglese, correggo